src/scripts/OSFramework/OSUI/Pattern/Progress/Circle/ProgressCircle.ts

```diff
@@ -44,27 +44,40 @@ namespace OSFramework.OSUI.Patterns.Progress.Circle {
 
 		// Convert progress value into offset to assign to our circle
 		private _progressToOffset(): void {
-			// Get the pattern parent size
-			const _elementSize =
-				this.selfElement.parentElement.clientHeight < this.selfElement.parentElement.clientWidth
-					? this.selfElement.parentElement.clientHeight
-					: this.selfElement.parentElement.clientWidth;
-
-			// Check the maxValue that the circle must have
-			if (this.selfElement.clientHeight < this.selfElement.parentElement.clientWidth) {
-				this._circletSize = this.selfElement.parentElement.clientWidth;
+			// Check wich size will be appliced on ProgressCircle
+			if (this.configs.ProgressCircleSize !== '') {
```

Comment:
This validation will create a strange behaviour with the following steps on the QA test screen:

1. Set width to something, like 100vw and apply. Works correctly ✅ 
2. Set width back to empty (which would be the default behavior of parent size). The size doesn't update, as the css variable isn't updated, due to this line.

Maybe the validation should be something like this:
`this.configs.ProgressCircleSize !== '' ||  this.configs.ProgressCircleSize === '' && Helper.Dom.Styles.GetStyleAttribute(this.selfElement,Enum.InlineStyleProp.ProgressCircleSize) !== '100%';`

Reply:
Fixed